main/busybox-initscripts/APKBUILD

```diff
@@ -12,12 +12,12 @@ makedepends=""
 install="$pkgname.post-install $pkgname.post-upgrade"
 subpackages=""
 options="!check"
-replaces="alpine-baselayout openntpd"
+replaces="alpine-baselayout openntpd openrc"
```

Comment:
@kaniini that's good enough, right?

Reply:
honestly i think i would prefer to see `depends="openrc>=[...]"` than using `replaces`.

it will be more robust.  the replaces then, should be unnecessary as openrc upgrade will always come before busybox-initscripts.

Reply:
Thanks for your input, done.